start.sh

```diff
@@ -62,6 +80,11 @@ echo "Starting Xvfb"
 Xvfb :0 -screen 0 1024x768x16 &
 echo "Launching wine64 V Rising"
 echo " "
-DISPLAY=:0.0 wine64 /mnt/vrising/server/VRisingServer.exe -persistentDataPath $p -serverName "$SERVERNAME" -saveName "$WORLDNAME" -logFile "$p/VRisingServer.log" "$game_port" "$query_port" 2>&1
-/usr/bin/tail -f /mnt/vrising/persistentdata/VRisingServer.log
+DISPLAY=:0.0 wine64 /mnt/vrising/server/VRisingServer.exe -persistentDataPath $p -serverName "$SERVERNAME" -saveName "$WORLDNAME" -logFile "$p/VRisingServer.log" "$game_port" "$query_port" 2>&1 &
+
```

Comment:
Not sure if its worth it, but can we have logs saved for last 3 runs?

like using `VRisingServer-$(date +'%Y%m%d-%H%M').log` (example: VRisingServer-20240513-2348.log)

We can maybe cleanup old logs when the script starts. This is helpful in case someone needs to go over old logs after starting a server. 

Scenario:
- User started server
- Realized that the server didn't exit correctly / Crashed
- Tries to figure out what happened by going over old logs.

Reply:
i dont think `VRisingServer.log` is ever wiped, every time the server starts it seems that it just writes to the end of the file, that said, it would be better if each run had their logs split by files, i will see what i can come up with.